### PR TITLE
Rename to DSCP

### PR DESCRIPTION
**Pull request by jonmattgray**
PR designed to merge shortly before/after renaming the repo to `dscp-ipfs`

Bump to `v2.0.0`

**Helm tests fail until rename because the workflow gets the directory (`helm/{REPO_NAME}/*`) from the GitHub repo name**

### CODE REVIEW

File: docker-compose.yaml
Comment (jonmattgray) [inline docker-compose.yaml:4]: need to wait until `vitalam-node` is renamed and built until we change or test workflow fails

Comment (jonmattgray) [reply docker-compose.yaml:4]: same applies to `helm-lint` and `helm-test` in workflows which point to the `node` e.g. `ct lint --validate-maintainers false --charts helm/${{ needs.repo_ids.outputs.repo_name }}/ --chart-repos vitalam-node=https://digicatapult.github.io/vitalam-node/`

**Review by mattdean-digicatapult — COMMENTED** [review]
lgtm once the docker and helm deps are updated

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**
